plugin.video.francetv/addon.xml

```diff
@@ -0,0 +1,39 @@
+<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
+<addon id="plugin.video.francetv" name="france.tv" version="2.0.0" provider-name="melmorabity">
+  <requires>
+    <import addon="xbmc.python" version="3.0.0"/>
+    <import addon="script.module.beautifulsoup4" version="4.8.2"/>
+    <import addon="script.module.inputstreamhelper" version="0.5.1"/>
+    <import addon="script.module.requests" version="2.22.0"/>
+  </requires>
+  <extension point="xbmc.python.pluginsource" library="addon_entry.py">
+    <provides>video</provides>
+  </extension>
+  <extension point="xbmc.addon.metadata">
+    <reuselanguageinvoker>true</reuselanguageinvoker>
+    <summary lang="en_GB">france.tv</summary>
+    <summary lang="fr_FR">france.tv</summary>
+    <description lang="en_GB">Live and catchup TV for France Télévisions channels</description>
+    <description lang="fr_FR">Direct et replay TV des chaînes France Télévisions</description>
+    <disclaimer lang="en_GB">Some streams may be georestricted to France</disclaimer>
+    <disclaimer lang="fr_FR">Certains contenus ne sont disponibles que depuis la France</disclaimer>
+    <platform>all</platform>
```

Comment:
You might want to add `<language>fr</language>` in the future so that your add-on can be hidden for foreign/non-french Kodi users if they desire (using the filters in the side bar of the repo browser)

Reply:
Good catch ;). Since france.tv may also broadcasts in other languages than French (in French overseas territories), I'll have to list all supported languages. But I keep it in mind ;)